src/base/bittorrent/torrenthandle.h

```diff
@@ -445,14 +445,26 @@ namespace BitTorrent
 
         QHash<QString, TrackerInfo> m_trackerInfos;
 
-        enum StartupState
+        enum ProcessingState
```

Comment:
I imagine I can leave this enum as it was (to represent only startup states, maybe just rename NotStarted to Preparing since I like it better) and add other boolean field to handle the state of temporarily resumed torrents. @Chocobo1, what do you say?

Reply:
Since I'm not so familiar with this part, I'm fine with anything that will work and is logical.

Reply:
I have two reasons for suggesting it.
1. The previous enum is more strict, because it describes the final process (it has an irrevocable state "Started"), so I like it more.
2. Since I won't touch on it, I'm not required to add comments there (although describing it is much easier than this modified one, so I might do that).